apps/server/src/modules/tool/launch-tool/service/strategy/basic-tool-launch.strategy.ts

```diff
@@ -24,7 +24,7 @@ export class BasicToolLaunchStrategy extends AbstractLaunchStrategy {
 		}
 
 		if (Object.keys(payload).length === 0) {
-			return '';
+			return undefined;
```

Comment:
It is fine if it is needed in this way. But in the most situations is a explizit null return saver then undefined.